### DISCUSSION TITLE
How can I give automatic public access to my clients with an api using npm private package service?

### DISCUSSION THREAD

**Discussion by eguvenc** [npm/feedback#1050] [Registry]
Hi,
I want to automatically grant access to my clients using the npm private package system.

Does npm private package subscription give me an API to do this? If not, is there another way to do this?

Is it enough to add collaborator to github repository via github api?

**Comment by ljharb** [comment]
No, github permissions are irrelevant. You can use `npm access` for this: https://docs.npmjs.com/cli/v7/commands/npm-access
